src/server/verification/faceRecognition/faceRecognitionHelper.js

```diff
@@ -87,11 +88,17 @@ const Helper = {
   },
 
   async isDuplicatesExist(zoomData: FormData, identifier: string) {
+    if (Config.allowFaceRecognitionDuplicates) {
+      log.info('NOTE: Skipping duplicates test')
```

Comment:
write the name of the function as first parameter in log

Reply:
fixed, commit 3a75013675688943af3b0e989cf2797881254c89, will be resolved once pushed